gc/base/GCExtensionsBase.hpp

```diff
@@ -84,6 +85,14 @@ class MM_RealtimeGC;
 class MM_VerboseManagerBase;
 struct J9Pool;
 
+namespace OMR {
+namespace GC {
+#if defined(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD)
+class HeapRegionStateTable;
+#endif /* defined(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD) */
+} // namespace OMR
+} // namespace GC
```

Comment:
Why not just include the header?

Reply:
As we talked about, including the VLHGC header in to GCExtensionsBase requires changing a lot of include paths throughout openj9.